packages/jaeger-ui/src/types/trace.tsx

```diff
@@ -52,16 +52,20 @@ export type SpanData = {
   startTime: number;
   duration: number;
   logs: Array<Log>;
-  tags: Array<KeyValuePair>;
-  references: Array<SpanReference>;
-  warnings: Array<string> | null;
+  process: Process;
```

Comment:
`process` is not part of the initial HTTP payload.

Reply:
Ha ok, so this could be part of Span, but not SpanData which represent the initial HTTP payload. Is that correct?